packages/integration-sdk-cli/src/commands/document.ts

```diff
@@ -146,11 +147,33 @@ function generateRelationshipTableFromAllStepEntityMetadata(
   return generated;
 }
 
+function generateMappedRelationshipTableFromAllStepEntityMetadata(
+  metadata: StepMappedRelationshipMetadata[],
+): string {
+  const generated = table([
+    [
+      'Source Entity `_type`',
+      'Relationship `_class`',
+      'Target Entity `_type`',
+      'Direction',
+    ],
+    ...metadata.map((v) => [
+      `\`${v.sourceType}\``,
+      `**${v._class}**`,
+      `\`*${v.targetType}*\``,
+      `${v.direction}`,
```

Comment:
Thoughts on the order and how this looks?

Reply:
I like it. The _italics_ are nice for the target.